api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -166,3 +175,42 @@ message Telemetry {
   //          returned.
   bytes frame_header = 6;
 }
+
+//----------------------------------------------------------------------------------------------
+// Scheduling APIs.
+//----------------------------------------------------------------------------------------------
+
+// Request for the `ListUpcomingAvailablePasses` method.
+message ListUpcomingAvailablePassesRequest {
+  // The ID of the satellite to list passes for. The ID of a satellite can be found on the
+  // StellarStation Console page for the satellite.
+  string satellite_id = 1;
+}
+
+// Response for the `ListUpcomingAvailablePasses` method.
+message ListUpcomingAvailablePassesResponse {
+  // The upcoming, schedulable passes for the satellite.
+  repeated Pass pass = 1;
+}
+
+// A pass during which a satellite can be communicated with from a given ground station.
+message Pass {
+  // A unique ID for this pass. This can be used in other APIs that accept `pass_id`.
+  string id = 1;
```

Comment:
I'd like to avoid giving users illusion that passes are identifiable objects. Even passes that are off by 1 seconds from each other have different ids, which may surprise users. Provided that `ReservePass` will be the only API that takes this value in the short term, I would make this explicitly a pass reservation token.